Formula/steem.rb

```diff
@@ -0,0 +1,33 @@
+class Steem < Formula
+  desc "Proof of Work blockchain with an unproven consensus algorithm."
+  homepage "https://steemit.com/"
+  url "https://github.com/steemit/steem.git",
```

Comment:
Unpegged submodules are irritating because it's yet another thing that can cause mysterious breakages that we then don't have much control over fixing. That's largely my concern about it.